authorization/index.md

```diff
@@ -187,18 +188,32 @@ cross-site request forgery or session fixation attacks.
 
       </td>
     </tr>
+     <tr>
+      <td><code>aud</code></td>
+      <td><span class="label label-default">recommended</span></td>
+      <td>
+
+URL of the EHR resource server from which the app wishes to retrieve a FHIR resource.  This parameter SHOULD be used to prevent presenting a bearer token to a counterfeit resource server (which then could use it to retrieve the authorized resource from the authentic resource server).  
+
+      </td>
+    </tr>
   </tbody>
 </table>
 
+The app MUST use an unpredictable value for the state parameter with at least 128 bits of entropy. The app MUST validate the value of the state parameter upon return to the redirect URL and MUST ensure that the state value is securely tied to the user’s current session (e.g., by relating the state value to a session identifier issued by the app).
+The app SHOULD limit the grants, scope, and period of time requested to the minimum necessary.   The app MAY digitally sign the request, with the EHR validating the signature.  
+
+If the app needs to authenticate the identity of the end-user, it should include two OpenID Connect scopes:  openid and profile.   When these scopes are requested, and the request is granted, the app will receive an id_token along with the access token.  For full details, see SMART launch context parameters (/authorization/scopes-and-launch-context).
```

Comment:
To create a relative link in markdown:

    [SMART launch context parameters](./scopes-and-launch-context)